google/cloud/spanner_v1/database.py

```diff
@@ -95,7 +96,14 @@ class Database(object):
 
```

Comment:
I think there should be a docs parameter here for `encryption_config`. This will also clarify the types question below.

Reply:
Done.